### PR TITLE
config for new charts

### PR DESCRIPTION
**Pull request by piyushmani**

### CODE REVIEW

File: app/chart/load-config.js [hunk 1]
```diff
@@ -10,32 +10,32 @@ var configs = {
     lineChartConfig: function (viz_id) {
 
         var chartconfigPromise = new Promise((resolve, reject) => {
-            
+
             try {
                 request(vizMetaApi + "/" + viz_id, function (error, response, body) {
-                    if(error){
+                    if (error) {
                         reject(error.message);
```
Comment (johibkhan2) [inline app/chart/load-config.js]: we can log the error before returning

File: app/chart/load-config.js [hunk 2]
```diff
@@ -107,9 +107,9 @@ var configs = {
 
             try {
                 request(vizMetaApi + "/" + viz_id, function (error, response, body) {
-                    if(error){
+                    if (error) {
                         reject(error.message);
```
Comment (johibkhan2) [inline app/chart/load-config.js]: log error here also

File: app/chart/load-config.js [hunk 3]
```diff
@@ -513,27 +513,256 @@ var configs = {
...
+
+    },
+    stackedHorizontalBarConfig: function (viz_id) {
+
+        var chartconfigPromise = new Promise((resolve, reject) => {
+
+            try {
+                request(vizMetaApi + "/" + viz_id, function (error, response, body) {
+                    if (error) {
+                        reject(error.message);
```
Comment (johibkhan2) [inline app/chart/load-config.js]: log it here as well

File: app/chart/load-config.js [hunk 4]
```diff
@@ -513,27 +513,256 @@ var configs = {
...
+                        result['kpiIconFontWeight'] = [];
+                        result['kpiIconColor'] = [];
+                        result['kpiIconExpression'] = [];
+                        result['FontSizeforDisplayName'] = [];
+                        for (var i = 0; i < measures.length; i++) {
+                            result['kpiDisplayName'].push(
+                                VisualizationUtils.getFieldPropertyValue(measures[i], 'Display name') ||
+                                result['measure'][i]
+                            );
+                            result['kpiAlignment'].push(VisualizationUtils.getFieldPropertyValue(measures[i], 'Text alignment'));
```
Comment (johibkhan2) [inline app/chart/load-config.js]: props "Text alignment" are coming from API so can we use that instead of hard coding them

Comment (piyushmani) [reply app/chart/load-config.js]: I have used the same format to extract chart info, which is used in all chart-service in flair-bi.

**Pull request merged**